sci-electronics/systemc/systemc-2.3.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit eutils toolchain-funcs flag-o-matic
+MY_P="${P}"
+
+DESCRIPTION="A C++ based modeling platform for VLSI and system-level co-design"
+HOMEPAGE="http://accellera.org/community/systemc"
+SRC_URI="http://www.accellera.org/images/downloads/standards/${PN}/${MY_P}.tar.gz"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="doc static-libs"
+KEYWORDS="~amd64 ~x86"
+
+RESTRICT="test"
+
+S="${WORKDIR}/${MY_P}"
+
+CUSTOM_CXX_STD="$(get-flag -std)"
+
+src_configure() {
+	# If user sets their own C++ standard, do not interfere
+	if [ -z "${CUSTOM_CXX_STD}" ]; then
+		append-cxxflags -std=c++14
+		append-cxxflags -DSC_CPLUSPLUS=201402L
+	fi
+	econf $(use_enable static-libs static) CXX=$(tc-getCXX) \
+	--with-unix-layout
+}
+
+src_install() {
+	dodoc AUTHORS ChangeLog INSTALL NEWS README RELEASENOTES
```

Comment:
`einstalldocs` instead because

1.  we don't want `INSTALL`
2. the defaults should cover it (consider whether you really need `RELEASENOTES`)